siac/walletcmd.go

```diff
@@ -507,10 +517,12 @@ func walletunlockcmd() {
 			return
 		}
 	}
-	password, err := speakeasy.Ask("Wallet password: ")
+	fmt.Print("Wallet password: ")
+	passwordBytes, err := terminal.ReadPassword(0)
 	if err != nil {
 		die("Reading password failed:", err)
 	}
+	password = string(passwordBytes)
 	qs := fmt.Sprintf("encryptionpassword=%s&dictonary=%s", password, "english")
```

Comment:
I think the cast is actually unnecessary here; `%s` should format a `[]byte` as a `string`